### DISCUSSION TITLE
NoPresentationContextException: No Presentation Context for Abstract Syntax: 1.2.840.10008.5.1.4.1.1.1

### DISCUSSION THREAD

**Discussion by netskill** [dcm4che/dcm4che#890] [Q&A] [answered]
**I got a exception as follow**
10:07:36,298 WARN  - DCM4CHEE<-GETSCU(2): Unable to perform sub-operation on association to GETSCU
org.dcm4che3.net.NoPresentationContextException: No Presentation Context for Abstract Syntax: 1.2.840.10008.5.1.4.1.1.1 - Computed Radiography Image Storage negotiated
        at org.dcm4che3.net.Association.pcFor(Association.java:897)
        at org.dcm4che3.net.Association.cstore(Association.java:925)
        at org.dcm4che3.net.service.BasicRetrieveTask.cstore(BasicRetrieveTask.java:258)
        at org.dcm4che3.net.service.BasicRetrieveTask.run(BasicRetrieveTask.java:185)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

**I runned some commands from  dcm4che-tool.As follow is the steps:**
1. Start SCP by command : dcmqrscp.bat --all-storage -b DCM4CHEE@localhost:11112 --dicomdir ./dcmqrscpServer/dicomInfos --filepath ./dcmqrscpServer/dicomFiles2/{00100020}/{0020000D}/{0020000E}/{00080018}.dcm

2.Run CStore to send dicom files  to SCP by cmd : storescu -c DCM4CHEE@localhost:11112 C:\libin\working\HDDSVN\302Test\302-DATA-CR-ALL\CR\group5\20100621\1719152

3. Run CGet to get dicom files from SCP by cmd : getscu.bat  -c DCM4CHEE@localhost:11112 -m StudyInstanceUID=1.2.840.31314.14143234.20100618074416.1719152 --directory ./getResult

**Expected behavior**
I hoped when the CGet command runned ,the dicom files can be read and save to local dir  ./getResult.
But i got a exception.

The dcm4che version is 5.22.6

**Comment by netskill** [comment]
I runed another test.
1 . Run CStore command to  send dicom files to dcm4chee ,the command : storescu -c DCM4CHEE@192.168.88.143:11112 C:\libin\working\HDDSVN\302Test\302-DATA-CR-ALL\CR\group5\20100621\1719157

2 . Run CGet command  to get dicom files from dcm4chee ,the command:getscu.bat  -c DCM4CHEE@192.168.88.143:11112 -m PatientID=100165783 -m StudyInstanceUID=1.2.840.31314.14143234.20100618074457.1719157 --directory ./getResult

It works fine, the CGet  can get dicom files and save them to local dir.
I want to know what is the difference between dcm4chee and dcm4che's dcmqrscp.

Do you have andy suggestion or did you have hit the issue?

**Reply by advanceMaster** [reply]
I encountered the same problem, which only occurs when I request getscp with other software. The getscu of DCM4CHE can request normally

**Comment by gunterze** [comment, marked as answer]
`dcmqrscp` does not support decompressing of compressed stored images. And by default `getscu` offers only one Presentation Context_ for _Computed Radiography Image Storage_ with _Transfer Syntax_ _Implicit VR Little Endian_ and _Explicit VR Little Endian_ - s.  [`$DCM4CHE_HOME/etc/getscu/store-tcs.properties`](https://github.com/dcm4che/dcm4che/blob/master/dcm4che-assembly/src/etc/getscu/store-tcs.properties#L9-L10)

**Reply by ddmer547** [reply]
If I want to expand, what should I do?

**Comment by netskill** [comment]
@gunterze Thanks!
As I know that the module DCM4CHEE-ARC can support the feature.
Except the module DCM4CHEE-ARC , are there any module can support the feature?

Many thanks!

**Comment by gunterze** [comment]
Alternatively you may explicitly specify the transfer syntax in which the CR image was received by `dcmqrscp` by option
```
    --store-tc <cuid:tsuid[(,|;)...]>     specifies negotiated Storage
                                          Transfer Capability. SOP Class
                                          and Transfer Syntaxes can be
                                          specified by its UID or its name
                                          in camel-Case (e.g.
                                          1.2.840.10008.5.1.4.1.1.2 or
                                          CTImageStorage). Semicolon
                                          separated Transfer Syntaxes will
                                          be offered in separate
                                          Presentation Contexts, where
                                          comma separated Transfer
                                          Syntaxes will be offered in one
                                          Presentation Context.
```
to be able to retrieve it by `getscu`.

**Reply by rafaelborbacs** [reply]
Hi, is there a way to make `dcmqrscp` accept and retrieve ANY dicom image with `getscu` no matter compressed or decompressed or anything else?
